docs/modules/policies/pages/conditions.adoc

```diff
@@ -99,7 +99,9 @@ condition:
         !(R.attr.qa == true || R.attr.canary == true))
 ----
 
-
+== Globals
+If your policy repeats the same expression in multiple conditions, you can declare it in the globals section of a policy and refer in the conditions to avoid a repeat.
+Please refer to xref:principal_policies.adoc[Principal policies] for a complete example.
```

Comment:
I liked the example you added to the resource policy doc in a previous commit. Can we add it here? I know it feels weird to repeat things in multiple places, but, clicking a link to look at an example tends to interrupt the mental flow of the reader and it's frustrating to switch back and forth between tabs when trying to understand something.